tests/test_examples.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+import os
+from pathlib import Path
+
+
+# Currently, these tests will run in the background if no screen is available
+# If you do have a screen available, plots will appear, and you'll need to
+# close them to make the tests pass.
+# TODO: see if we can make these run locally w/o showing matplotlib
+# @patch("matplotlib.pyplot.show")
+def test_examples():
+    scripts = [
+        "2D_QLIPP_simulation/2D_QLIPP_forward.py",
+        "2D_QLIPP_simulation/2D_QLIPP_recon.py",
+        "3D_PODT_phase_simulation/3D_PODT_Phase_forward.py",
+        "3D_PODT_phase_simulation/3D_PODT_Phase_recon.py",
+        "PTI_simulation/PTI_Simulation_Forward_2D3D.py",
+        "PTI_simulation/PTI_Simulation_Recon2D.py",
+        "PTI_simulation/PTI_Simulation_Recon3D.py",
+    ]
+
+    for script in scripts:
+        path = os.path.join(os.getcwd(), "examples/", script)
+        completed_process = subprocess.run(["python", path])
```

Comment:
Another way is to have a main function for these scripts and call them in separate test cases. That way if a subset fails it's easier to debug. But that may impair readability by a little bit?

Reply:
Thanks for the suggestion...I've cleaned this up a bit. 

FYI some of these scripts are dependent on the earlier scripts (the recons depend on the forward simulations), which is why I've grouped them into three groups.